python-package/xgboost/spark/core.py

```diff
@@ -473,6 +473,15 @@ def _validate_params(self) -> None:
 
         self._validate_gpu_params()
 
+    def _run_on_gpu(self) -> bool:
+        """If train or transform on the gpu according to the parameters"""
+
+        return (
+            use_cuda(self.getOrDefault(self.device))
+            or self.getOrDefault(self.use_gpu)
+            or self.getOrDefault(self.getParam("tree_method")) == "gpu_hist"
+        )
```

Comment:
Did we support the `gpu_hist` with Pyspark in 1.7? I thought the `use_gpu` was the only way to get GPU support with PySpark even before 2.0.

Reply:
Yes, pyspark always supports gpu_hist

Reply:
Do you want gpu_hist is not supported in pyspark?